Assets/Scripts/Models/Furniture.cs

```diff
@@ -613,6 +621,26 @@ public void CancelJobs()
         }
     }
 
+    public void EnqueueJobs()
```

Comment:
I would call these functions PauseJobs and ResumeJobs, since that is essentially what they are for.